Daf.Meta.Editor/ViewModels/HubRelationshipsViewModel.cs

```diff
@@ -51,20 +55,28 @@ public ObservableCollection<HubRelationshipViewModel> HubRelationships
 			}
 		}
 
-		private DataSource? _selectedDataSource;
+		private DataSourceViewModel? _selectedDataSource;
 
-		public DataSource? SelectedDataSource
+		public DataSourceViewModel? SelectedDataSource
 		{
 			get => _selectedDataSource;
 			set
 			{
 				SetProperty(ref _selectedDataSource, value);
+
+				//if (value != null)
+				//{
+				//	HubRelationships = new(value.DataSource.HubRelationships.Select(hubRelationship => new HubRelationshipViewModel(hubRelationship)));
+				//}
+
+				//OnPropertyChanged(nameof(StagingColumns));
```

Comment:
Remove commented-out code. Either that, or uncomment and fix. :)

Reply:
Fixed.